mentoris/urls.py

```diff
@@ -29,4 +29,6 @@
     path("main/", views.main, name="main"),
     path("main/<int:volume_id>/", views.main, name="main_vol_chap"),
     path("main/<int:volume_id>/<chapter_id>/", views.chapter, name="chapter"),
+    path('download_pdf/<int:blob_key>/', views.download_pdf, name='download_pdf'),
```

Comment:
Not really a big deal but you're using single quotes when everything else is double